CONTRIBUTING.md

```diff
@@ -341,16 +341,8 @@ I love it! Very clean code and great tests. lgtm.
 
 ## Contact
 
-The Kata Containers community can be reached through its dedicated IRC
-channels, Slack channels, and mailing lists:
-
-* IRC:
-  * Development discussions: `#kata-dev @ freenode.net`.
-  * General discussions: `#kata-general @ freenode.net`.
-
-* [Slack channels](https://katacontainers.slack.com/) ([invite](http://bit.ly/KataSlack)).
-
-* [Mailing lists](http://lists.katacontainers.io/).
+The Kata Containers community can be reached
+[through various channels](#README.md#join-us).
```

Comment:
I don't think this double # link is working for me - I suspect you might not need the first # ?

Reply:
Ta - fixed!